python/graphstorm/inference/np_infer.py

```diff
@@ -17,6 +17,7 @@
 """
 import time
 import logging
+import dgl
```

Comment:
DGL is a 3rd part import should go in the second import group

Reply:
changed.